app/domain/vp2015/export/tabular/course_reporting/aggregation.rb

```diff
@@ -59,58 +90,9 @@ def kursart_label(kursart)
           end
         end
 
-        def attributes_of_leistungskategorie # rubocop:disable Metrics/MethodLength This is more data than function
-          attrs = [
-            :anzahl_kurse,
-            :kursdauer,
-
-            :teilnehmende,
-            :teilnehmende_behinderte,
-            :teilnehmende_angehoerige,
-            :teilnehmende_weitere
-          ]
-
-          attrs += if treffpunkt?
-                     [:total_stunden_betreuung]
-                   else
-                     [
-                       :total_absenzen,
-                       :absenzen_behinderte,
-                       :absenzen_angehoerige,
-                       :absenzen_weitere,
-
-                       :total_tage_teilnehmende,
-                       :tage_behinderte,
-                       :tage_angehoerige,
-                       :tage_weitere
-                     ]
-                   end
-
-          attrs += [
-            :betreuende,
-            :leiterinnen,
-            :fachpersonen,
-            :hilfspersonal_ohne_honorar,
-            :hilfspersonal_mit_honorar
-          ]
-
-          attrs += [:kuechenpersonal] unless treffpunkt?
-
-          attrs += [
-            :direkter_aufwand,
-            :honorare_inkl_sozialversicherung,
-            :unterkunft,
-            :uebriges,
-
-            :direkte_kosten_pro_le,
-            :total_vollkosten,
-            :vollkosten_pro_le,
-            :beitraege_teilnehmende,
-            :betreuungsschluessel,
-            :anzahl_spezielle_unterkunft
-          ]
-
-          attrs
+        def attributes_of_leistungskategorie
+          TP_ATTRIBUTES if treffpunkt?
```

Comment:
```suggestion
          return TP_ATTRIBUTES if treffpunkt?
```

Ich denke, dass das explizite `return` hier fehlt.